tests/loader_layer_tests.cpp

```diff
@@ -336,10 +622,9 @@ TEST_F(OverrideMetaLayer, NewerComponentLayerInMetaLayer) {
         inst.create_info.set_api_version(1, 1, 0);
         inst.CheckCreate();
         VkPhysicalDevice phys_dev = inst.GetPhysDev();
-        const char* err_message =
-            "verify_meta_layer_component_layers: Meta-layer uses API version 1.1, but component layer 0 uses API version 1.2.  "
-            "Skipping this layer.";
-        ASSERT_TRUE(env->debug_log.find(err_message));
+        ASSERT_TRUE(
+            env->debug_log.find("verify_meta_layer_component_layers: Meta-layer uses API version 1.1, but component layer 0 uses "
```

Comment:
Should we not include the function name in all of these in case we ever move the warning around?

Reply:
Yes, because if you update where the warning lives (ie move it to a different function) but don't update the message, that is a misleading message.